task/slack-webhook-notification/0.1/README.md

```diff
@@ -8,4 +8,6 @@ Sends message to slack using incoming webhook
 |message|Message to be sent||true|
 |secret-name|Secret with least one key where value is webhook URL for slack. eg. oc create secret generic my-secret --from-literal team1=https://hooks.slack.com/services/XXX/XXXXXX --from-literal team2=https://hooks.slack.com/services/YYY/YYYYYY |slack-webhook-notification-secret|false|
 |key-name|Key in the key in secret which contains webhook URL for slack.||true|
+|files|List of file to dump. The content will be added to the message.|[]|false|
+|submodules|List of submodules name to dump. Git log since previous submodule commit will be added to the message. The previous submodule commit is found by looking at the previous commit in the repository that declares the submodules.|[]|false|
```

Comment:
nitpick: it's better to update this in the yaml so that the `generate-readme.sh` script doesn't remove this the next time someone remembers to use it

Reply:
done